Library/Homebrew/cmd/fetch.rb

```diff
@@ -32,6 +32,11 @@ def fetch
   def fetch_bottle? f
     return true if ARGV.force_bottle? && f.bottle
     return false unless f.bottle && f.pour_bottle?
+    if formula.file_modified?
+      ohai "#{formula.full_name} is modified; fetching source"
+      ohai "Fetch the bottle with --force-bottle"
```

Comment:
Using a `puts` means we're not restricted by length, so maybe even "To fetch the bottle instead, run with --force-bottle"?